satellizer.js

```diff
@@ -786,42 +785,10 @@
         return normalize(joined);
       };
     })
-    .factory('satellizer.storage', ['satellizer.config', function(config) {
-      switch (config.storage) {
-        case 'localStorage':
-          if ('localStorage' in window && window['localStorage'] !== null) {
-            return {
-              get: function(key) { return localStorage.getItem(key); },
-              set: function(key, value) { return localStorage.setItem(key, value); },
-              remove: function(key) { return localStorage.removeItem(key); }
-            };
-          } else {
-            console.warn('Warning: Local Storage is disabled or unavailable. Satellizer will not work correctly.');
-            return {
-              get: function(key) { return undefined; },
-              set: function(key, value) { return undefined; },
-              remove: function(key) { return undefined; }
-            };
-          }
-          break;
-
-        case 'sessionStorage':
-          if ('sessionStorage' in window && window['sessionStorage'] !== null) {
-            return {
-              get: function(key) { return sessionStorage.getItem(key); },
-              set: function(key, value) { return sessionStorage.setItem(key, value); },
-              remove: function(key) { return sessionStorage.removeItem(key); }
-            };
-          } else {
-            console.warn('Warning: Session Storage is disabled or unavailable. Satellizer will not work correctly.');
-            return {
-              get: function(key) { return undefined; },
-              set: function(key, value) { return undefined; },
-              remove: function(key) { return undefined; }
-            };
-          }
-          break;
-      }
+    .factory('satellizer.storage', ['satellizer.config', 'localStorageService', function(config, localStorageService) {
```

Comment:
Given `satellizer.storage` and `localStorageService` use the same interface (`get`, `set`, `remove`), you can remove this service. 
If you want to keep the service `satellizer.storage` and restrict his scope, you can provide such service:

```
.factory('satellizer.storage', ['localStorageService', function(localStorageService) {
  return {
    get: localStorageService.get,
    set: localStorageService.set,
    remove: localStorageService.remove,
  };
}
```